project/CrossJava.scala

```diff
@@ -100,7 +100,7 @@ object CrossJava {
   }
 
   object JavaDiscoverConfig {
-    private val JavaHomeDir = """(java-|jdk-?)(1\.)?([0-9]+).*""".r
+    private val JavaHomeDir = """(java-|jdk-?|adoptopenjdk-)(1\.)?([0-9]+).*""".r
```

Comment:
on OSX I have an `open-jdk-11+28`, do we need to add such prefix too?

Reply:
I think there is special support for Jabba elsewhere

Reply:
There's another regex further down in the file for the jabba dirs which is more lenient in the first place.

Reply:
thanks